lib/icons/sync.js

```diff
@@ -0,0 +1,21 @@
+
+/** @jsx element */
+
+import element from 'dekujs/virtual-element';
+
+export default function render(color) {
+  return (
+    <svg width="100%" height="100%" viewBox="0 0 18 18">
+      <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g transform="translate(-690.000000, -1450.000000)" stroke="#3CC76A" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round">
```

Comment:
You can use `color` in the `stroke` attribute here. Feel free to set a default for this if you need it. 

`function render(color = '#3CC76A') {}`